crates/sui-graphql-rpc/src/types/balance_change.rs

```diff
@@ -22,15 +22,16 @@ impl BalanceChange {
         use NativeOwner as O;
 
         match self.stored.owner {
-            O::AddressOwner(addr) | O::ObjectOwner(addr) => Some(Owner {
+            O::AddressOwner(addr)
+            | O::ObjectOwner(addr)
+            // TODO-DNS do we want consensus coins included in balance here?
```

Comment:
This will include consensus managed coins, yep, but that seems like the right thing to me.

Reply:
"DNS" whats that mean in this context?

Reply:
>from the google days

just outed myself that during my google tenure i never contributed to google3 🙃